tools/pot/openvino/tools/pot/graph/passes.py

```diff
@@ -874,3 +876,32 @@ def find_shape_subgraph_endpoints(out_ports: List[Port], visited: set = None) ->
             visited_nodes.add(in_port.node)
         visited.add(in_port)
     return visited_nodes
+
+def remove_converts(graph: Graph):
+    for op in graph.get_op_nodes(type='Convert'):
+        source_op = op.in_port(0).get_source().node
+        if source_op.type == 'Const':
+            # Get access to data node between Const and Convert operations and set Insert_Convert_operation_after
+            # to restore Convert operation
+            source_op.out_node(0)['Insert_Convert_operation_after'] = True
```

Comment:
We have a precision check in the backward transformation but we don't have it in this one. It can lead to some problems in the future when we support other precisions. I propose doing the explicit check here as well before marking the node by attribute.

Reply:
Check added.